lte/gateway/python/integ_tests/s1aptests/test_attach_detach_dedicated_looped.py

```diff
@@ -0,0 +1,102 @@
+"""
+Copyright (c) 2016-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+"""
+
+import unittest
+import s1ap_types
+import time
+
+from integ_tests.s1aptests import s1ap_wrapper
+from integ_tests.s1aptests.s1ap_utils import SpgwUtil
+
+
+class TestAttachDetachDedicatedLooped(unittest.TestCase):
+
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self._spgw_util = SpgwUtil()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_attach_detach(self):
+        """ attach/detach + dedicated bearer test in loop with a single UE """
+        num_ues = 1
+        loop = 10
```

Comment:
10 is too long for the test
lets loop 3 times only

Reply:
done